addons/http_routing/models/ir_http.py

```diff
@@ -376,7 +376,7 @@ def _geoip_setup_resolver(cls):
 
     @classmethod
     def _geoip_resolve(cls):
-        if 'geoip' not in request.session:
+        if not request.session.get('geoip', False):
```

Comment:
Useless default value of `get`

```suggestion
        if not request.session.get("geoip"):
```